### ISSUE TITLE
`~` not parsed correctly in path arguments

### ISSUE THREAD

**Issue by foxyseta** [ulyssa/iamb#173]
Some commands (such as `:upload`) require path arguments. Although `.`/`..` are fully supported already, the same does not apply for `~`. I think it should both (1) work properly when the command is issued and (2) not confuse the autocompleter.

**Comment by ptman** [comment]
https://github.com/ulyssa/iamb/pull/538 ?

**Comment by VAWVAW** [comment]
No, #538 only works for config values. The same would need to be implemented for commands. But I feel like `modalkit` would be the better place to implement it.

**Comment by foxyseta** [comment]
Indeed, I can still reproduce this on `iamb 0.0.12-alpha.2 (fc1c164)`.

**Comment by ulyssa** [comment]
~This should be resolved now that ulyssa/modalkit#186 is merged and pulled into iamb in #664.~

Never mind, I misunderstood your comment before.
